### PR TITLE
docs: use precise file names for public and private key files

### PR DESCRIPTION
**Pull request by franku**

### CODE REVIEW

File: docs/manuals/source/TasksAndConcepts/DataEncryption.rst [hunk 1]
```diff
@@ -69,35 +69,34 @@ The various algorithms are exposed via an entirely re-usable, OpenSSL-agnostic A
 Generating Private/Public Encryption Keys
 -----------------------------------------
 
-:index:`\ <single: Encryption; Generating Private/Public Encryption Keypairs>`\ 
+:index:`\ <single: Encryption; Generating Private/Public Encryption Keypairs>`\
 
-Generate a Master Key Pair with:
+Generate a public/private key-pair with:
```
Comment (pstorz) [inline docs/manuals/source/TasksAndConcepts/DataEncryption.rst:74]: Master public/private key pair

File: docs/manuals/source/TasksAndConcepts/DataEncryption.rst [hunk 2]
```diff
@@ -69,35 +69,34 @@ The various algorithms are exposed via an entirely re-usable, OpenSSL-agnostic A
...
 
 
 
 ::
 
-     openssl genrsa -out master.key 2048
-     openssl req -new -key master.key -x509 -out master.cert
+     openssl genrsa -out master.priv.key 2048
+     openssl req -new -key master.priv.key -x509 -out master.publ.key
```
Comment (pstorz) [inline docs/manuals/source/TasksAndConcepts/DataEncryption.rst:81]: I would call it "pub", not "publ", as this is usually the name in ssh like id_dsa.pub

File: docs/manuals/source/TasksAndConcepts/DataEncryption.rst (hunk 1 above)
Comment (franku) [inline docs/manuals/source/TasksAndConcepts/DataEncryption.rst:74]:
```suggestion
Generate a Master public/private key-pair with:
```

File: docs/manuals/source/TasksAndConcepts/DataEncryption.rst (hunk 2 above)
Comment (franku) [inline docs/manuals/source/TasksAndConcepts/DataEncryption.rst:81]:
```suggestion
     openssl req -new -key master.priv.key -x509 -out master.pub.key
```

File: docs/manuals/source/include/config/FdClientPki.conf [hunk 3]
```diff
@@ -2,9 +2,9 @@ FileDaemon {
...
    # encryption configuration
-   PKI Signatures = Yes                          # Enable Data Signing
-   PKI Encryption = Yes                          # Enable Data Encryption
-   PKI Keypair    = "/etc/bareos/example-fd.pem" # Public and Private Keys
-   PKI Master Key = "/etc/bareos/master.cert"    # ONLY the Public Key
-   PKI Cipher     = aes128                       # specify desired PKI Cipher here
+   PKI Signatures = Yes                           # Enable Data Signing
+   PKI Encryption = Yes                           # Enable Data Encryption
+   PKI Keypair    = "/etc/bareos/example-fd.pem"  # Public and Private Keys in one file
+   PKI Master Key = "/etc/bareos/master.publ.key" # ONLY the Public Key
```
Comment (franku) [inline docs/manuals/source/include/config/FdClientPki.conf:8]:
```suggestion
   PKI Master Key = "/etc/bareos/master.pub.key" # ONLY the Public Key
```

File: docs/manuals/source/TasksAndConcepts/DataEncryption.rst [hunk 4]
```diff
@@ -69,35 +69,34 @@ The various algorithms are exposed via an entirely re-usable, OpenSSL-agnostic A
...
 
 
 ::
 
-     openssl genrsa -out example-fd.key 2048
-     openssl req -new -key example-fd.key -x509 -out example-fd.cert
-     cat example-fd.key example-fd.cert >example-fd.pem
+     openssl genrsa -out example-fd.priv.key 2048
+     openssl req -new -key example-fd.priv.key -x509 -out example-fd.publ.key
+     cat example-fd.priv.key example-fd.publ.key >example-fd.pem
```
Comment (franku) [inline docs/manuals/source/TasksAndConcepts/DataEncryption.rst:93]:
```suggestion
     cat example-fd.priv.key example-fd.pub.key >example-fd.pem
```

File: docs/manuals/source/TasksAndConcepts/DataEncryption.rst [hunk 5]
```diff
@@ -69,35 +69,34 @@ The various algorithms are exposed via an entirely re-usable, OpenSSL-agnostic A
...
-public and a private key.
+Please note the extensions given to these key-files. For example, a .pem file can contain all the following: private keys (RSA and DSA), public keys (RSA and DSA) and (x509) certificates. It is the default format for OpenSSL. It stores data Base64 encoded DER format, surrounded by ASCII headers, so is suitable for text mode transfers between systems. A .pem file may contain any number of keys either public or private. We use it in cases where there is both, a public and a private key.
 
-Above we have used the .cert extension to refer to X509 certificate encoding that contains only a single public key.
+Above we have used the .publ.key extension to refer to X509 certificate encoding that contains only a single public key.
```
Comment (franku) [inline docs/manuals/source/TasksAndConcepts/DataEncryption.rst:99]:
```suggestion
Above we have used the .pub.key extension to refer to X509 certificate encoding that contains only a single public key.
```

**Pull request merged**